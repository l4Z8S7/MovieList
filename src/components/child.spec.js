import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Child from './child';

describe('test child', () => {
	test('test child render', () => {
		const wrapper = shallow(<Child />);
		expect(toJson(wrapper)).toMatchSnapshot();
	})
})