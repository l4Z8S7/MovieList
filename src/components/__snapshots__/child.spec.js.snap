// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test child test child render 1`] = `
<div>
  Child
</div>
`;

exports[`test child test render 1`] = `
<div>
  Child
</div>
`;

exports[`test index test render 1`] = `
<div>
  Child
</div>
`;
